frontend/internal-packages/agent/src/chat/workflow/nodes/designSchemaNode.ts

```diff
@@ -198,7 +198,11 @@ export async function designSchemaNode(
   // Clear retry flags after processing
   const finalResult = {
     ...result,
-    messages: [...messages, invokeResult.value.message],
+    messages: [
+      ...state.messages,
+      new HumanMessage(userMessage),
+      invokeResult.value.message,
+    ],
```

Comment:
Corresponds to the same as line 172

https://github.com/liam-hq/liam/blob/26fbb969d3d49689be4b751a621a40f4022c05a0/frontend/internal-packages/agent/src/chat/workflow/nodes/designSchemaNode.ts#L171-L172